app/uk/gov/hmrc/securemessage/services/SecureMessageService.scala

```diff
@@ -26,12 +26,21 @@ import scala.concurrent.{ ExecutionContext, Future }
 @SuppressWarnings(Array("org.wartremover.warts.ImplicitParameter"))
 class SecureMessageService @Inject()(conversationRepository: ConversationRepository) {
 
-  def getConversations(enrolment: Enrolment)(implicit ec: ExecutionContext): Future[List[ConversationDetails]] = {
+  def getConversations(enrolment: Enrolment)(implicit ec: ExecutionContext): Future[List[ConversationMetaData]] = {
     val enrolmentToIdentifier = Identifier(enrolment.name, enrolment.value, Some(enrolment.key))
     conversationRepository.getConversations(enrolment).map { coreConversations =>
       coreConversations.map(conversations =>
-        ConversationDetails.coreToConversationDetails(conversations, enrolmentToIdentifier))
+        ConversationMetaData.coreToConversationMetadata(conversations, enrolmentToIdentifier))
     }
   }
 
+  def getConversation(client: String, conversationId: String, enrolment: Enrolment)(
```

Comment:
getConversations and getConversation need to be unit tested (we need a SecureMessageServiceSpec). If you can't do it as part of this PR, let me know and I'll do it after you merge.

Reply:
creating some service level tests now :+1: